spec/controllers/v0/efolder_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
can this be moved to a request spec instead of a controller spec?  That's the recommended approach by the rails and rspec core teams